config/database.yml

```diff
@@ -17,6 +17,7 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: omer.katzir
```

Comment:
this makes the whole project very specific to your environment, I would lose the username here.
If you need it for the tests, make sure to modify the `test` line.